src/sentry/static/sentry/app/views/stream.jsx

```diff
@@ -531,6 +531,9 @@ const Stream = createReactClass({
   },
 
   onLatestContextChange(context) {
+    // Don't do anything unless environment is changing
+    if (context.environment === this.state.activeEnvironment) return;
```

Comment:
this implies that the only context change we care about ever is environment, is that true?

Reply:
Yep. In this view you can't modify project or organization so we don't have to listen for it (with the exception of starring the project and we want to ignore that)